src/main/start-main-application/lens-window/application-window/create-electron-window.injectable.ts

```diff
@@ -76,9 +76,7 @@ const createElectronWindowInjectable = getInjectable({
         webPreferences: {
           nodeIntegration: true,
           nodeIntegrationInSubFrames: true,
-          webviewTag: true,
```

Comment:
Instead of removing this line should we keep it with `false` value (to make it clear)?

Reply:
Well th default is false and I don't see electron changing that default now.